resources/web/styles.pcss

```diff
@@ -198,19 +199,6 @@ p.remark {
     clear: right;
 }
 
-/* Images */
```

Comment:
This is what I started out doing. But there wasn't an example of a screenshot in the readme and on thing led to another and now we have this.